contentgrid-spring-boot-starter/build.gradle

```
@@ -7,6 +7,7 @@ dependencies {
     api platform(project(':contentgrid-spring-boot-platform'))
     api "com.contentgrid.thunx:thunx-api-spring-boot-starter"
 
+    api project(':contentgrid-automations-rest')
```

```suggestion
    implementation project(':contentgrid-automations-rest')
```

This project does not deliver any APIs that we want to expose to the user when they add a dependency on the `contentgrid-spring-boot-starter`